src/dbally/views/pandas_base.py

```diff
@@ -36,6 +36,15 @@ async def apply_filters(self, filters: IQLQuery) -> None:
         """
         self._filter_mask = await self.build_filter_node(filters.root)
 
+    async def apply_aggregation(self, aggregation: IQLQuery) -> None:
+        """
+        Applies the aggregation of choice to the view.
+
+        Args:
+            aggregation: IQLQuery object representing the aggregation to apply
+        """
+        pass  # pylint: disable=unnecessary-pass
```

Comment:
Pandas can support aggregations. Maybe at least mark it with TODO and add a ticket?

BTW, why you chose disabling the pylint rule instead of changing the code to be inline with it?